CHANGELOG.md

```diff
@@ -9,6 +9,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfluff/sqlfluff/releases) and will be copied in here on each release (please remember to update the issues and contributors to links!). There is no need to manually edit this file going forward.
 -->
 
+## [0.8.0] - 2021-11-06
+
+## What’s Changed
+
+This minor release brings an improvement to the performance of the parser, and a rebuild of the Jinja Templater. Lots of dialect improvements have also been done. Full list of changes below:
+
+## 🚀 Enhancements
+
+* Added trigger support in ANSI - and extended it in Postgres [#1818](https://github.com/sqlfluff/sqlfluff/pull/1818) [@WittierDinosaur](https://github.com/WittierDinosaur)
+* Exasol: Make references more strict [#1829](https://github.com/sqlfluff/sqlfluff/pull/1829) [@sti0](https://github.com/sti0)
+* Hive: fixed hive insert statement [#1828](https://github.com/sqlfluff/sqlfluff/pull/1828) [@mifercre](https://github.com/mifercre)
+* Hive: fixed hive create temporary external table [#1825](https://github.com/sqlfluff/sqlfluff/pull/1825) [@mifercre](https://github.com/mifercre)
```

Comment:
```suggestion
* Hive: INSERT statement support [#1828](https://github.com/sqlfluff/sqlfluff/pull/1828) [@mifercre](https://github.com/mifercre)
```

Will move the temp external table to bugs fix section.

(I've edited the issues too so Release notes will be updated)